bitnami/postgresql-ha/templates/_helpers.tpl

```diff
@@ -580,9 +580,7 @@ Compile all warnings into a single message, and call fail.
 {{- define "postgresql-ha.validateValues.nodesHostnames" -}}
 {{- $postgresqlFullname := include "postgresql-ha.postgresql" . }}
 {{- $postgresqlHeadlessServiceName := printf "%s-headless" (include "postgresql-ha.postgresql" .) }}
-{{- $releaseNamespace := .Release.Namespace }}
-{{- $clusterDomain:= .Values.clusterDomain }}
-{{- $nodeHostname := printf "%s-00.%s.%s.svc.%s:1234" $postgresqlFullname $postgresqlHeadlessServiceName $releaseNamespace $clusterDomain }}
+{{- $nodeHostname := printf "%s-00.%s:1234" $postgresqlFullname $postgresqlHeadlessServiceName }}
```

Comment:
This was a typo in the existing code
```suggestion
{{- $nodeHostname := printf "%s-00.%s:5432" $postgresqlFullname $postgresqlHeadlessServiceName }}
```
It doesn't matter for the validation since the length is going to be the same, but just to fix the typo

Reply:
Also, for what it's worth, based on my testing, the `:5432` is not part of the hostname length. It is stored in a separate parameter. See the pgpool log output from #5394

Reply:
I've removed the port from the length check.